src/main/java/net/minecraftforge/event/world/AddBuiltinRegistryEvent.java

```diff
@@ -0,0 +1,33 @@
+package net.minecraftforge.event.world;
+
+import com.mojang.serialization.Codec;
+import net.minecraft.core.Registry;
+import net.minecraft.core.RegistryAccess;
+import net.minecraft.resources.ResourceKey;
+import net.minecraftforge.eventbus.api.Event;
+import net.minecraftforge.fml.event.IModBusEvent;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.function.BiConsumer;
+
+public class AddBuiltinRegistryEvent extends Event implements IModBusEvent
```

Comment:
This whole event class needs some serious documentation.

Reply:
I hope, I haven't missed anything important in the new documentation. Is there anything that I should add?